package.json

```diff
@@ -188,5 +188,8 @@
     "not op_mini all"
   ],
   "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e",
-  "dependencies": {}
+  "dependencies": {},
+  "resolutions": {
```

Comment:
Using canvas 3.1 is much better for bun compatibility and running in newer node, and is only used for unit tests.